downloads/swan-lake-release-notes/swan-lake-2201.5.0/RELEASE_NOTE.md

```diff
@@ -0,0 +1,494 @@
+---
+layout: ballerina-left-nav-release-notes
+title: 2201.5.0 (Swan Lake) 
+permalink: /downloads/swan-lake-release-notes/2201.5.0/
+active: 2201.5.0
+redirect_from: 
+    - /downloads/swan-lake-release-notes/2201.5.0
+    - /downloads/swan-lake-release-notes/2201.5.0-swan-lake/
+    - /downloads/swan-lake-release-notes/2201.5.0-swan-lake
+    - /downloads/swan-lake-release-notes/
+    - /downloads/swan-lake-release-notes
+---
+
+## Overview of Ballerina Swan Lake 2201.5.0
+
+<em>2201.5.0 (Swan Lake Update 5) is the fifth update release of Ballerina Swan Lake, and it includes a new set of features and significant improvements to the compiler, runtime, standard library, and developer tooling. It is based on the 2022R4 version of the Language Specification.</em> 
+
+## Update Ballerina
+
+Run either of the commands below to directly update to 2201.5.0 using the [Ballerina Update Tool](/learn/cli-documentation/update-tool/).
```

Comment:
Needs to be updated now since we are mentioning only one.

Reply:
Will be fixed in https://github.com/ballerina-platform/ballerina-dev-website/pull/6740.